doc/install/aspect-setup_stampede.sh

```diff
@@ -31,12 +29,12 @@ case $1 in
 
     "trilinos")
         # Build Trilinos
-     if [ ! -f $HOME/Downloads/trilinos-11.12.1-Source.tar ]; then
-            echo "You need trilinos-11.12.1-Source.tar in ~/Downloads"
+     if [ ! -f $HOME/Downloads/trilinos-11.12.1-Source.tar.gz ]; then
+            echo "You need trilinos-11.12.1-Source.tar.gz in ~/Downloads"
             exit 1
```

Comment:
We started with the tar file because there were issues with uncompressing .gz using the script.